src/main/kotlin/net/maizegenetics/phgv2/cli/ExportHvcf.kt

```diff
@@ -0,0 +1,61 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.util.stream.Collectors
+
+/**
+ * This will export the give samples dataset to a hvcf file.
+ */
+class ExportHvcf : CliktCommand() {
+
+    private val myLogger = LogManager.getLogger(ExportHvcf::class.java)
+
+    val dbpath by option(help = "Folder name where TileDB datasets are stored")
+        .required()
+
+    val sampleNames by option(help = "Sample names to export")
+        .required()
+
+    val outputDir by option("-o", "--outputDir", help = "Directory where temporary and final files will be written")
+        .required()
+
+    override fun run() {
+
+        // tiledbvcf export --uri tiledb/hvcf_dataset -O z --sample-names Ref --output-dir exported-vcfs
+
+        val builder = ProcessBuilder(
```

Comment:
Consider calling VariantLoadingUtils.bgzipAndIndexGVCFfile() to do the gzipping and indexing.

Reply:
I didn't know about that. Good catch. Changed.

Reply:
@tcasstevens I'm coming to this late - has bgzipAndIndexGVCFFile() been called, but that isn't committed yet?

Reply:
Yes that has been committed.